hack/boilerplate/test/pass.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
I don't know but I think those files keep the original date since is just a copy from the other repo

Reply:
addressed, i also bring back the original date of related file.